src/cartservice/tests/CartServiceTests.cs

```diff
@@ -90,20 +110,24 @@ public async Task AddItem_ItemExists_Updated()
             Assert.Equal(2, cart.Items[0].Quantity);
 
             // Cleanup
-            await client.EmptyCartAsync(new EmptyCartRequest{ UserId = userId });
+            await client.EmptyCartAsync(new EmptyCartRequest { UserId = userId });
         }
 
         [Fact]
         public async Task AddItem_New_Inserted()
         {
-            string userId = Guid.NewGuid().ToString();
+            // Setup test server and client
+            using var server = await _host.StartAsync();
```

Comment:
I notice a few lines here (the server/client setup) are duplicated between tests, do you think it's worth writing a helper function or okay to leave as-is?

Reply:
I would suggest keeping it as-is to keep top-to-bottom readability clear. I'd be happy migrate to a private method if preferred.

Reply:
Sounds good!